crates/ln-dlc-node/src/node/ln_channel.rs

```diff
@@ -77,6 +79,18 @@ impl<P> Node<P> {
                         )
                         .map_err(|e| anyhow!("Could not force close channel {e:?}"))
                 } else {
+                    if let Some(SubChannel { state, .. }) = self.list_dlc_channels()?.first() {
+                        if *state != SubChannelState::OffChainClosed {
```

Comment:
I think it makes sense although I haven't tested it yet.

Can we make sure the app user gets clear feedback on why it failed? The logs don't cut it for me.

Reply:
I could add it, but figured, its just for us at the moment so why bother. If you think its important, I can add it.

Reply:
> 

we haven't yet implemented any user feedback mechanism, have we? is the idea to port it from PoC?

Reply:
> I could add it, but figured, its just for us at the moment so why bother. If you think its important, I can add it.

Fair enough, you can ignore it if it's just for us!

Reply:
> we haven't yet implemented any user feedback mechanism, have we? is the idea to port it from PoC?

That's true. I tried something very simple that seemed to work well with https://github.com/get10101/10101/pull/533 (which wasn't merged), so we could go in that direction.

Reply:
I've seen that. It was very nice. We should come back to something like that eventually.